docs/zh-CN/docs/sql-manual/sql-functions/string-functions/elt.md

```diff
@@ -51,7 +51,7 @@ mysql> select elt(0, 'aaa', 'bbb');
 +----------------------+
 | NULL                 |
 +----------------------+
-mysql> select elt(2, 'aaa', 'bbb');
+mysql> select elt(3, 'aaa', 'bbb');
```

Comment:
`en docs` also has the same problem as `zh-CN docs`, please also fix it.

Reply:
ok,resubmit